server_common/channel_access.py

```diff
@@ -82,6 +93,15 @@ class AlarmStatus(Enum):
 
 
 class ChannelAccess(object):
+    # Create a thread poll so that threads are reused and so ca contexts that each thread gets are shared. This also
+    # caps the number of ca library threads. 20 is chosen as being probably enough but limited.
+    thread_pool = ThreadPoolExecutor(max_workers=NUMBER_OF_CAPUT_THREADS)
```

Comment:
Is it worth setting the "thread_name_prefix" argument too to potentially help future debugging?